src/model/person/APersonalInfoPerson.java

```diff
@@ -305,4 +307,49 @@ public Role getRole(){
 
     @Override
     public Optional<HairColorTrait> getPreferredHair() { return this.preferredHair; }
+
+	@Override
+	public Map<IPerson, IRelationship> getRelationships() {
+		return relationships;
+	}
+
+	@Override
+	public void addRelationship(IPerson other, IRelationship relationship) {
+		this.relationships.put(other, relationship);
+	}
+
+	@Override
+	public void meet(IPerson other, int year) {
+		if (!this.relationships.containsKey(other)) {
+			IRelationship relationship = new RelationshipImpl(this, other, year);
+			this.addRelationship(other, relationship);
+			other.addRelationship(this, relationship);
+		}
+	}
+
+	@Override
+	public boolean knows(IPerson other) {
+		return this.relationships.containsKey(other);
+	}
+
+	@Override
+	public IPerson getSignificantOther() {
+		return this.significantOther;
+	}
+
+	@Override
+	public void setSignificantOther(IPerson other) {
+		this.significantOther = other;
+	}
+
+	@Override
+	public boolean hasSignificantOther() {
+		return this.significantOther != null;
+	}
+
+	@Override
+	public int compareTo(IPerson other) {
+		return this.age - other.getAge();
```

Comment:
I'm a little uneasy having our compareTo be based solely on age. Wherever this is used, could we pass a different comparator?

Reply:
I think I can take this out for now, because it didn't get used where I was thinking of using it.